### ISSUE TITLE
engineImageLinkTXD function incompatitible with engineRequestModel

### ISSUE THREAD

**Issue by JeViCo** [multitheftauto/mtasa-blue#3187]
### Describe the bug

I tried using both IMG functions & `engineRequestModel` and failed. I tried using `engineImageLink*` functions like `engineImportTXD` and `engineReplaceModel`:
```lua
local imgElement = engineLoadIMG("img/admiral.img")
engineAddImage(imgElement)

local saModelId = getVehicleModelFromName("admiral")
local reqModelId = engineRequestModel("vehicle", saModelId)

engineImageLinkTXD(imgElement, "admiral1.txd", reqModelId)
engineImageLinkDFF(imgElement, "admiral1.dff", reqModelId)
```
Textures didn't load so i followed the wiki and changed code to
```lua
local imgElement = engineLoadIMG("img/admiral.img")
engineAddImage(imgElement)

local saModelId = getVehicleModelFromName("admiral")
local reqModelId = engineRequestModel("vehicle", saModelId)

local txdId = engineGetModelTXDID(reqModelId)
local dffId = reqModelId
engineImageLinkTXD(imgElement, "admiral1.txd", txdId)
engineImageLinkDFF(imgElement, "admiral1.dff", dffId)
```
I don't know why does it require additional API calls instead of direct id pass but it works.

TL;DR; after doing some researches i've managed to freeze client. I requested id for two vehicles with same `parentId`. I noticed that `engineGetModelTXDID` returned same value for both ids. I attempted to change vehicles's model with one of requested ids and my client froze.

### Steps to reproduce

1. Create two or more ids with `engineRequestModel` function with same `parentId`
2. Link all ids to IMG
3. Try passing any requested id to `setElementModel` function

OR

1. Use sample resource
[img_test.zip](https://github.com/multitheftauto/mtasa-blue/files/12645549/img_test.zip)

### Version

Client: v1.6-release-22204 (Windows 10, 64 bit)

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by JeViCo** [comment]
I also discovered `engineRequestTXD` function but the syntax is completely different from `engineRequestModel`. It also might have conflicts within 2 IMG archives when using same name. In short, it doesn't seem to fit the current case.

**Comment by Fernando-A-Rocha** [comment]
@TheNormalnij 🫡

**Comment by Xenius97** [comment]
Use
https://wiki.multitheftauto.com/wiki/EngineRequestTXD and textures will work.

**Comment by JeViCo** [comment]
> Use https://wiki.multitheftauto.com/wiki/EngineRequestTXD and textures will work.

Isn't it for existing TXDs?

**Comment by JeViCo** [comment]
I've made some tests around [EngineRequestTXD](https://wiki.multitheftauto.com/wiki/EngineRequestTXD) and it actually works. Just use different string every time you need a fresh TXD id.

Personally, I'm not a big fan of those additional ID changes as this adds additional complexity and general confusion of how things work
